06-building-a-program-start-to-finish/automating-tasks/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: Gulp
+- **Deliver**: a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files
+- **With**: *independently*
+
+#### Targeted Learning Outcomes
+- [Output for Gulp automation](https://www.youtube.com/watch?v=7ffZj81sxnc)
```

Comment:
I have now done another video that covers HTML, CSS and JS automation using Gulp.